LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
Regarding the changes, these were done automatically by the IDE, i just thought there was no harm in updating in other modules too.

Regarding the multi-module, I can see it useful to reduce duplication of the gradle wrapper for example, and also the fact that it's easier to load it immediately from the root, having access to all the projects.

This is just my point of view, I don't remember the historical reasons behind this repo structure honestly.